src/ast/NumericConstant.h

```diff
@@ -10,11 +10,12 @@
  *
  * @file NumericConstant.h
  *
+ * Defines the numeric constant class
+ *
  ***********************************************************************/
 
 #pragma once
 
-#include "RamTypes.h"
```

Comment:
This is needed for `Ramsigned`

Reply:
I don't like this dependency. It would be great to have a declaration in TypeDomain.h in ast. 
This should be done in a later fix (i.e. ast and ram should be decoupled).